src/components/Profile/styles.scss

```diff
@@ -0,0 +1,22 @@
+.flex-row {
```

Comment:
CSS classes should describe what some component is: a class could be e.g. title, title icon, description, box-header etc